tika-core/src/main/java/org/apache/tika/pipes/PipesServer.java

```diff
@@ -455,33 +455,33 @@ private Fetcher getFetcher(FetchEmitTuple t) {
         }
     }
 
-    protected MetadataListAndEmbeddedBytes parseFromTuple(FetchEmitTuple t, Fetcher fetcher) {
-        FetchKey fetchKey = t.getFetchKey();
+    protected MetadataListAndEmbeddedBytes parseFromTuple(FetchEmitTuple fetchEmitTuple, Fetcher fetcher) {
+        FetchKey fetchKey = fetchEmitTuple.getFetchKey();
+        Metadata fetchResponseMetadata = new Metadata();
```

Comment:
The metadata that goes in the fetchemittuple was envisioned to be user-injected metadata that passed through the parse process and was emitted (provenance metadata).

I think we need to put both metadatas on the fetchemittuple.

Reply:
@nddipiazza any chance you can revert this in main so that we have a working build? Thank you!

Reply:
shoot i didn't realize i was deplying broken builds! reverted. i'll make this change and make a new pr